app/controllers/applicants_controller.rb

```diff
@@ -1,6 +1,6 @@
 class ApplicantsController < ApplicationController
   def index
-    @applicants = Applicant.all
+    @applicants = Applicant.includes(:positions).where('positions.id' => params[:position_id])
```

Comment:
Check for the existence of params now and if there are no params return Applicant.all. Will need to figure out why the controller isn't returning only associated objects in a future iteration.

```
      user = FactoryGirl.create(:user)
      applicant = Applicant.create
      Applicant.create
      position = Position.create(applicants: [applicant])
      get positions_path, headers: { 'Content-Type' => 'application/vnd.api+json', 'Accept' => 'application/vnd.api+json', 'authorization' => "Token token=#{user.authentication_token}, email=#{user.email}" }
      get position_applicants_path(position_id: position.id), headers: { 'Content-Type' => 'application/vnd.api+json', 'Accept' => 'application/vnd.api+json', 'authorization' => "Token token=#{user.authentication_token}, email=#{user.email}" }
      puts JSON.parse(response.body)
```

Is a test that should return only associated objects but returns all of them with Applicant.all